types/wallet.d.ts

```diff
@@ -21,13 +22,121 @@ declare module 'wallet' {
       ): Promise<TransactionResult>;
     }
 
+    interface MultiSigWallet extends ContractBase {
+      owners(): Promise<Address[]>;
```

Comment:
The correct autogenerated getter looks like `owners(num: AnyNumber): Promise<Address>`